### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bl0**
Add Unsupervised Feature Learning via Non-parameteric Instance Discrimination

### CODE REVIEW

**Comment by nbro** [comment]
@bl0 I read the abstract of that paper. How is this paper related to self-supervised learning? As far as I understood, it is related to unsupervised learning.

**Comment by bl0** [comment]
This paper learns to discriminate between individual instances. In other words, the ground truth in the training is the index of the image instead of the semantic label. So I think it's a self-supervised learning.

And the author said in the paper:
> we may end up with a representation that captures apparent similarity among instances, just like how class-wise supervised learning still retains apparent similarity among classes. This formulation
of unsupervised learning as instance-level discrimination is also technically appealing, as it could benefit from the latest advances in discriminative supervised learning, e.g. on new network architectures.

**Comment by jason718** [comment]
@bl0 @nbro yes it is. They also compared with other papers. 

Self-supervised is a little bit confusing, so many people have asked me the difference between it can unsupervised-learning.

Normally, people would use unsupervised to refer 'clustering,...', and use 'self-supervised' to refer "free-supervision from data"

**Comment by bl0** [comment]
Yes, I think the difference between self-supervised learning and unsupervised learning is the training strategy: The training process of self-supervised learning just like supervised learning with some automatically obtained target, while the unsupervised learning cannot do any backpropagation.
